### DISCUSSION TITLE
feature_mortgage

### DISCUSSION THREAD

**Discussion by ryanhtafesa** [ryanhtafesa/T24S2T1EXTG04-REV2#1] [Announcements]
feature_mortage discussion!

**Comment by ryanhtafesa** [comment]
Hi @AlexandraKoshur 

I will start the re-development of the mortgage calculator again - if that's OK?
I will let you know on here my progress later tonight

**Comment by ryanhtafesa** [comment]
Hi @AlexandraKoshur 

The repo is ready for you to clone. 
I have created my feature branch, it is working and ready to merge, but I have not merged yet, pending your ability to clone the current development. 

Once you're on a roll, let me know when you've completed your branch and pushed back to GitHub.
We need to follow the original plan we had - according to KT. 
So I will merge Mortgage Calculator, then we can merge currency converter.

**Reply by AlexandraKoshur** [reply]
@ryanhtafesa, I have created mu CurencyConverter and pushed it back to GitHub. Please note - I did not add the navigation, please let me know if you want me to add it.

**Reply by ryanhtafesa** [reply]
Thanks! - I mentioned in the other link I think we cannot avoid the merge conflict based on our current plan. 
Hopefully you feel OK to try tackle it once you merge. I did one the other day in practice, and it was OK to do, just have to ensure you choose to add above or below so we keep both lines of code we created.
